src/selectors/getProductsByCategory.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-vars */
+import { results } from "../mocks/en-us/products.json";
+
+export const getProductsByCategoryId = (categories = []) => {
+  if (categories.length === 0) {
```

Comment:
An easy way of doing this is 
```suggestion
  if (categories.length === 0) return results;
  const products = [];
  / / everything else
  return products
```
I try avoiding else statements as much as I can because its easier to read, in this case only when the categories is empty you want to make something different